source/includes/_simulator-guide.html.md

```diff
@@ -0,0 +1,130 @@
+    - Interactive
+* EnergyPlus (via BCVTB)
+
+A list of examples using these coordinators can be found on the [Examples][7] page.
+
+# Running a Simulator Locally
+
+> Simulator config help text
+
+```
+$ python <simulator_file>.py --help
+usage: hw-cartpole.py [-h] [--accesskey ACCESSKEY] [--username USERNAME]
+                      [--url URL] [--proxy PROXY] [--brain BRAIN]
+                      [--predict [PREDICT]] [--verbose VERBOSE]
+                      [--performance PERFORMANCE]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --accesskey ACCESSKEY
+                        The access key to use when connecting to the BRAIN
+                        server. If specified, it will be used instead of any
+                        access key information stored in a bonsai config file.
+  --username USERNAME   Bonsai username.
+  --url URL             Bonsai server URL. The URL should be of the form
+                        "https://api.bons.ai"
+  --proxy PROXY         Proxy server address and port. Example: localhost:3128
+  --brain BRAIN         The name of the BRAIN to connect to. Unless a version
+                        is specified the BRAIN will connect for training.
+  --predict [PREDICT]   If set, the BRAIN will connect for prediction with the
+                        specified version. May be a positive integer number or
+                        'latest' for the most recent version. For example:
+                        --predict=latest or --predict=3
+  --verbose VERBOSE     Enables logging. Alias for --log=all
+  --performance PERFORMANCE
+                        Enables time delta logging. Alias for --log=perf.all
+```
+
+During development, you may need to iterate quickly on your simulation, reward function, and lesson plans. You may also need to connect a debugger, or do additional logging to help you get your simulation ready to train. The Bonsai SDK provides a set of capabilities to help the developer iterate quickly on their simulation while running from a local laptop or desktop machine.
+
+The `Config` object has a built in command line parser which can be used to override the local running environment. It comes with a set of standard switches which can be viewed with the `--help` option. The current default configuration is read from environment variables, a `~/.bonsai` configuration file in the user’s home directory, a `./.bonsai` and a `./.brains` configuration file in the current working directory, command line options, and programmatically in code.
+
+During the course of normal operation, one should only need to specify the prediction/training mode on the command line. The rest of the options will be read from configuration files.
+
+One is not limited to launching a single simulator locally. If you have the compute resources, each new simulator will open a new connection to the server and add to the training pool. You can do this on a per process basis, or at the thread level.
```

Comment:
suggest not mentioning threads here, adding a full example later.

Reply:
To speed up training, you can run multiple simulators in parallel. See `<link to below section>` for more.